### PR TITLE
Support HTTP api for browser recevier

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/browser/provider/handler/rest/BrowserPerfDataReportServletHandler.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        this.config = config;
+        this.perfDataListenerManager = perfDataListenerManager;
+
+        MetricsCreator metricsCreator = moduleManager.find(TelemetryModule.NAME)
+                                                     .provider()
+                                                     .getService(MetricsCreator.class);
+
+        perfHistogram = metricsCreator.createHistogramMetric(
+            "browser_perf_data_in_latency", "The process latency of browser performance data",
+            new MetricsTag.Keys("protocol"), new MetricsTag.Values("grpc")
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/browser/provider/handler/rest/BrowserPerfDataReportServletHandler.java:61]: I think this tag is not right.

Comment (arugal) [reply oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/browser/provider/handler/rest/BrowserPerfDataReportServletHandler.java]: updated.

File: docs/en/protocols/Browser-HTTP-API-Protocol.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# HTTP API Protocol
+
+HTTP API Protocol defines the API data format, including api request and response data format.
+They use the HTTP1.1 wrapper of the official [SkyWalking Browser Protocol](Browser-Protocol.md). Read it for more details.
+
+## Performance Data Report
+
+Detail information about data format can be found in [BrowserPerf.proto](https://github.com/apache/skywalking-data-collect-protocol/blob/master/Browser/BrowserPerf.proto).
```
Comment (wu-sheng) [inline docs/en/protocols/Browser-HTTP-API-Protocol.md:8]: 404 link

Comment (arugal) [reply docs/en/protocols/Browser-HTTP-API-Protocol.md:8]: Sorry, should be`browser`.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**